src/coreclr/jit/importer.cpp

```diff
@@ -14150,6 +14282,32 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         op1 = gtNewCastNodeL(type, op1, uns, lclTyp);
                     }
                     else
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks